src/sssom/constants.py

```diff
@@ -6,14 +6,14 @@
 from functools import cached_property, lru_cache
 from typing import Any, Dict, List, Literal, Set
 
-import pkg_resources
+import importlib_resources
```

Comment:
This is the context right: https://setuptools.pypa.io/en/latest/pkg_resources.html

![image](https://github.com/mapping-commons/sssom-py/assets/7070631/3af2b1c2-f0af-439a-a62e-724c6a70d630)

Reply:
Yes!

Reply:
Thx